pages/model/[uuid].tsx

```diff
@@ -404,10 +434,18 @@ function Model() {
 
             <Box sx={{ mb: 4 }} />
 
+            <ConfirmationDialogue
+              showConfirmationDialogue={deleteConfirmOpen}
+              onCancel={handleDeleteCancel}
+              onConfirm={handleDeleteConfirm}
+              confirmationModalTitle='Delete the model?'
```

Comment:
I think a better title would be simply `Delete Model`. The dialog content is already being used to ask the user a question, we don't need to ask them twice :)